tests/functional/lifecycle/LifecycleTask.js

```diff
@@ -328,10 +334,10 @@ class ProducerMock {
 
     sendToTopic(topicName, entries, cb) {
         const entry = JSON.parse(entries[0].message);
-        if (topicName === 'bucket-tasks') {
+        if (topicName === bucketTasksTopic) {
             this.sendCount.bucket++;
             this.entries.bucket.push(entry);
-        } else if (topicName === 'object-tasks') {
+        } else if (topicName === objectTasksTopic) {
```

Comment:
shoudl we also count entries pushed to transition topic ?

Reply:
No need as nothing is pushed into it here, only ones pushing to it are either sorbet or the replication process